pkgs/development/python-modules/taxi/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, appdirs
+, requests
+, click
+, setuptools
+, backends ? [ ]
```

Comment:
pythonModuels should not have options due to problems when multiple variants of the same package end up in one package.

Reply:
Answered below.